src/cavendish_particle_tracks/_magnification_dialog.py

```diff
@@ -161,23 +161,38 @@ def _on_click_magnification(self) -> None:
         self.table.setItem(0, 0, QTableWidgetItem(str(self.a)))
         self.table.setItem(0, 1, QTableWidgetItem(str(self.b)))
 
+    def _activate_cal_layer(self):
```

Comment:
```suggestion
    def _activate_calibration_layer(self):
```

Reply:
And where is this called? Nowhere?

Reply:
Next time you call the Magnification in the main widget:

```python
def _on_click_magnification(self) -> MagnificationDialog:
        """When the 'Calculate magnification' button is clicked, open the magnification dialog"""

        dlg = MagnificationDialog(self)
        dlg.show()
        if self.mag_dlg is not None:
            self.mag_dlg.show()
            self.mag_dlg.raise_()
            self.mag_dlg._activate_cal_layer()
            return self.mag_dlg
       ...
```